server/test/services/zwave/lib/nodesExpectedResult.json

```diff
@@ -1770,7 +1771,8 @@
         "unit": "celsius",
         "has_feedback": true,
         "min": 0,
-        "max": 0
+        "max": 0,
+        "step": 1
```

Comment:
There are still "step" attribute in this file

Reply:
Fixed